chain-test/src/test/scala/org/bitcoins/chain/blockchain/sync/ChainSyncTest.scala

```diff
@@ -0,0 +1,49 @@
+package org.bitcoins.chain.blockchain.sync
+
+import akka.actor.ActorSystem
+import org.bitcoins.chain.api.ChainApi
+import org.bitcoins.chain.util.{BitcoindChainHandlerViaRpc, ChainUnitTest}
+import org.bitcoins.core.crypto.DoubleSha256DigestBE
+import org.scalatest.FutureOutcome
+
+import scala.concurrent.Future
+
+class ChainSyncTest extends ChainUnitTest {
+  override type FixtureParam = BitcoindChainHandlerViaRpc
+
+  override implicit val system = ActorSystem(s"chain-sync-test-${System.currentTimeMillis()}")
```

Comment:
Not strictly related to this PR, but we could move this into `ChainUnitTest` and do `ActorSystem(s"${getClass.getSimpleName}-${System.currentTimeMillis()}")`